pkg/kritis/cryptolib/authenticated_attestation.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cryptolib
+
+import (
+	"encoding/json"
+
+	"github.com/pkg/errors"
+)
+
+// AtomicContainerSig represents a JSON-encoded Attestation payload. It
```

Comment:
This seems to match `authenticatedAttestation` well. Should we expose this and use this in `verifier` instead of using `ImageName` and `ImageDigest` separately?